lib/commons/aria/is-accessible-ref.js

```diff
@@ -1,13 +1,24 @@
 /* global aria, axe, dom */
-function findDomNode(node, functor) {
-	if (functor(node)) {
-		return node;
+const idRefsRegex = /^idrefs?$/;
+
+function cacheIdRefs(node, refAttrs) {
+	if (node.hasAttribute) {
+		if (node.nodeName.toUpperCase() === 'LABEL' && node.hasAttribute('for')) {
+			axe._cache.idRefs[node.getAttribute('for')] = 1;
+		}
+
+		refAttrs
+			.filter(attr => node.hasAttribute(attr))
+			.forEach(attr => {
+				const attrValue = node.getAttribute(attr);
+				axe.utils.tokenList(attrValue).forEach(id => {
+					axe._cache.idRefs[id] = 1;
```

Comment:
Is there a particular reason you're setting this to a number, rather than setting a boolean?

Reply:
Nope, just a force of habit. Can change to a boolean.